replication_report-extensions.Rmd

```diff
@@ -0,0 +1,361 @@
+
+```{r echo = FALSE, warning = FALSE}
+
+# Load packages:
+suppressPackageStartupMessages(require(dplyr))
+suppressPackageStartupMessages(require(tidyr))
+suppressPackageStartupMessages(require(pwr)) 
+suppressPackageStartupMessages(require(broom))
+suppressPackageStartupMessages(require(lsr))
+suppressPackageStartupMessages(require(ggplot2))
+suppressPackageStartupMessages(require(MBESS))
+suppressPackageStartupMessages(require(psych))
+suppressPackageStartupMessages(require(car))
+suppressPackageStartupMessages(require(MASS))
+suppressPackageStartupMessages(require(stargazer))
+
+# Derrive table of frequencies:
+Liked_Music <- c("21(57%)","16(43%)", "N=72, X^2(1)=8.59, p=0.003, phi=0.35")
+Disliked_Music <- c("8(23%)","27(77%)", "")
+Condition <- c("AdvertisedPen", "NonAdvertisedPen", "Chi-Square")
+Freq_Table = cbind(Condition, Liked_Music, Disliked_Music)
+stargazer(Freq_Table, type = "text", title = "Study 3: Frequencies of Pen Choice for Each Conditon & Chi-Square")
+
+```
+
+### (2.2) Explanation.
+The table above is a reproduction of the frequencies and percentages of subjects' choice for advertised and non-advertised pen in the Liked Music & Disliked Music conditions, respectively, in Study 3. Note that at the bottom of the table is also the chi-square statistic, p-value and effect size (phi) for the data, indicating a medium-sized effect of music condition on pen choice. The table also demonstrates that the frequency of choice in the Disliked Music condition was in the hypothesized direction: Subjects tended to choose the non-advertised pen when made to listen to disliked music in an advertisement. Please note that the data represented above follows the exclusion criteria set forth in the published article. Specifically, any subjects who had participated in a prior version of the study were excluded (two subjects). In addition, 19 subjects were excluded for their so-called "deviant music evaluations". These subjects were foundd to either somewhat dislike the "liked music" (music attiude score below 3) or somewhat like the "disliked music" (music attitude score above 3).
+
+### (2.3) Code for analyses & table.
+Note: the 'eval = FALSE' parameter was added to this code chunk to prevent printing of command outputs superfluous to the results reported in the original paper.
+
+```{r}
+
+# Load data
+dat <- read.csv("GORN_study3.csv", header = TRUE)
+dat <- tbl_df(dat)
+
+# Designate factors:
+dat$Groep <- as.factor(dat$Groep)
+dat$music <- as.factor(dat$music)
+dat$chose_advertised_pen <- as.factor(dat$chose_advertised_pen)
+
+# Filter out 2 excluded participants who accidentily took study twice in the paper:
+dat_e <- dat %>%
+  filter(excluded == 0)
+
+## Analyses on ENTIRE sample:
+
+# Test whether music condition is independent of pen choice:
+# "music"" variable: (1 == liked music, 2 == disliked music)
+# "chose_advertised_pen" variable: (0 == not chosen, 1 == chosen)
```

Comment:
this is much more clear.  People don't need a codebook to understand this or what its values mean